nf_core/modules/remove.py

```diff
@@ -58,5 +58,27 @@ def remove(self, module):
 
         log.info("Removing {}".format(module))
 
+        # Remove entry from modules.json
+        if not self.remove_modules_json_entry(module):
+            return False
+
         # Remove the module
         return self.clear_module_dir(module_name=module, module_dir=module_dir)
+
+    def remove_modules_json_entry(self, module):
+        # Load 'modules.json'
+        modules_json_path = os.path.join(self.dir, "modules.json")
+        with open(modules_json_path, "r") as fh:
+            modules_json = json.load(fh)
```

Comment:
We should probably verify that the `modules.json` and catch the exception if it doesn't to still remove the module. Just to be sure, for the initial phase ... maybe some people won't have the file. Although I think that the transition will be really fast.

Reply:
I thought about it, but didn't add it at the end since we check for the file in `has_valid_pipeline`. But I guess it doesn't do any harm checking for it, and if the code is changed at some point it might cause errors.